tilgangskontroll/src/main/kotlin/no/nav/amt/tiltak/tilgangskontroll/autentisering/NavAnsattAutentiseringController.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.amt.tiltak.tilgangskontroll.autentisering
+
+import no.nav.amt.tiltak.common.auth.AuthService
+import no.nav.amt.tiltak.common.auth.Issuer
+import no.nav.amt.tiltak.core.port.VeilederService
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/api/nav-ansatt/autentisering")
+open class NavAnsattAutentiseringController(
+	private val authService: AuthService,
+	private val veilederService: VeilederService,
+) {
+
+	@ProtectedWithClaims(issuer = Issuer.AZURE_AD)
+	@GetMapping("/meg")
+	fun me(): MegDto {
```

Comment:
"meg"? Klarer ikke bestemme meg for om jeg liker det eller ikke

Reply:
Kanskje litt rart navn ja, hadde ikke hatt noe imot å kalle det noe annet heller